src/lib/project.js

```diff
@@ -286,6 +273,78 @@ function kebabCase(str) {
   return str.toLowerCase().replace(' ', '-');
 }
 
+function scaffoldSvelte() {
+  // `-y` installs the latest version of create-svelte without prompting.
+  spawnSync('npm', ['create', 'svelte@latest', '-y', 'ui'], {
+    stdio: 'inherit',
+    shell: true,
+  });
+  sh.cp(
+    path.join(__dirname, 'ui', 'svelte', 'hooks.server.js'),
+    path.join('ui', 'src')
+  );
+}
+
+function scaffoldNext() {
+  // https://nextjs.org/docs/api-reference/create-next-app#options
+  spawnSync('npx', ['create-next-app@latest', 'ui', '--use-npm'], {
+    stdio: 'inherit',
+    shell: true,
+  });
+  sh.rm('-rf', path.join('ui', '.git')); // Remove NextJS' .git; we will init .git in our monorepo's root.
+  // Read in the NextJS config file and add the middleware.
+  const nextConfig = fs.readFileSync(path.join('ui', 'next.config.js'), 'utf8');
+  const newNextConfig = nextConfig.replace(
+    /^}(.*?)$/gm, // Search for the last '}' in the file.
+    `
```

Comment:
If the standard generated nextConfig is:

```
/** @type {import('next').NextConfig} */
const nextConfig = {
  reactStrictMode: true,
  swcMinify: true,
}

module.exports = nextConfig
```

would this `const newNextConfig = nextConfig.replace(
    /^}(.*?)$/gm, // Search for the last '}' in the file.` remove the `}` under the `swcMinify: true,` line - ie the closing bracket of the nextConfig object, or am I reading this wrong?